Parser/parser.py

```diff
@@ -202,6 +217,18 @@ def Compile(code:list) -> str:
                 code_string += node.visit() + ";\n"
             
             case NodeTypes.IF_STMT:
-                pass
+                node = parse_IfStmt(line)
+                code_string += node.visit() + "{\n"
```

Comment:
Each AstNode drive class have its own 'visit()' method. Instead of adding characters like ';\n' and '{\n' here is not good idea you should add this in AstNode drives classes

Reply:
Well you are right.